providers/wallet.csv

```diff
@@ -46,6 +46,16 @@ c98wallet,C98wallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,FA
 frontier,Frontier,TRUE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,FALSE,FALSE,TRUE,FALSE,TRUE,TRUE,FALSE,"[""ERC-20"", ""BEP-20"", ""ERC-721""]",TRUE,$0,"[""x"", ""discord"", ""telegram""]",[],"[""EN"",""FR"",""KO"",""ZH"", ""TR"", ""TH"", ""VI""]",FALSE,"[""[Website](https://www.frontier.xyz/)""]"
 ctrl-wallet,Ctrl Wallet,TRUE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,FALSE,FALSE,TRUE,TRUE,TRUE,TRUE,TRUE,"[""ERC-20"", ""ERC-721"", ""ERC-1155"", ""BEP-20""]",TRUE,Paid,"[""chatbot""]","[""audit by third parties""]","[""EN""]",FALSE,"[""[Website](https://ctrl.xyz/)""]"
 wallet3,Wallet 3,TRUE,"[""iOS"",""Android""]",FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,"[""ERC-4337""]",FALSE,$0,"[""discord""]","[""audit by third parties""]","[""EN""]",FALSE,"[""[Website](https://wallet3.io/)""]"
+cwallet,Cwallet,FALSE,"[""Browser Extension"",""iOS"",""Android""]",FALSE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,"[""ERC-20"",""BEP-2"",""BEP-20"",""ERC-721"",""SPL"",""TRC-20"",""TRC-10"",""BRC-20""]",TRUE,$0 ,"[""email"",""24/7 live chat"",""x"",""discord"",""telegram"", ""webform""]","[""audit by third parties""]","[""EN"", ""24 Others""]",TRUE,"[""[Website](https://cwallet.com/)""]"
```

Comment:
Starred column should always be false as per style guide (https://github.com/Chain-Love/chain-love/wiki/Common-Columns#starred)